script/_nat64

```diff
@@ -32,20 +32,23 @@
 
 TAYGA_DEFAULT=/etc/default/tayga
 TAYGA_CONF=/etc/tayga.conf
+TAYGA_IPV4_ADDR=192.168.255.1
 TAYGA_IPV6_ADDR=fdaa:bb:1::1
 NAT44_SERVICE=/etc/init.d/otbr-nat44
 WLAN_IFNAMES=eth0
 
 # Currently only ubuntu is verified.
 #
-without NAT64 || test $PLATFORM = ubuntu || die "nat64 is not tested under $PLATFORM."
+without NAT64 || test $PLATFORM = ubuntu || test $PLATFORM = raspbian || die "nat64 is not tested under $PLATFORM."
 
 nat64_install()
 {
     with NAT64 || return 0
 
     test -f $TAYGA_DEFAULT -a -f $TAYGA_CONF || die 'Cannot find tayga configuration file!'
     sudo sed -i 's/^RUN="no"/RUN="yes"/' $TAYGA_DEFAULT
+    sudo sed -i 's/^IPV4_TUN_ADDR=""/IPV4_TUN_ADDR="'$TAYGA_IPV4_ADDR'"/' $TAYGA_DEFAULT
```

Comment:
Are this change necessary because the configuration file already set these values?

Reply:
Yes, this is necessary, without those two address are not assigned to nat64 interface, then DNS64 doesn't work.